### DISCUSSION TITLE
Please use tags for versioning instead of commit messages

### DISCUSSION THREAD

**Discussion by geocine** [c-smile/sciter-js-sdk#199] [Ideas]
Please use tags for versioning instead of commit messages. There are number of benefits
- Can easily find versions we need via tags and the releases page. Don't need to grep the commits and find the hash of the version we are looking for
- Can create releases based on tags and add patch notes or whatever you want to put there including additional binaries you might want added
- Can easily direct users to download older version of sciter-js-sdk if support on their bindings is lagging behind.

Or you can do both if you prefer it on commit messages.

**Comment by c-smile** [comment]
Yeah, makes sense. Done.
